cmake/options.cmake

```diff
@@ -68,13 +68,15 @@ option (WITH_VIRTIO_DEVICE "Build with virtio device (back end)  enabled" ON)
 option (WITH_VIRTIO_MASTER "Build with virtio driver (front end)  enabled" ON)
 option (WITH_VIRTIO_SLAVE "Build with virtio device (back end)  enabled" ON)
 
+set (CMAKE_ERROR_DEPRECATED OFF)
```

Comment:
I don't think it's a good idea, if OS's like Zephyr decide to treat depreciation as an error, we shouldn't silence the error. 
I guess they want to be up to date by using this mechanism....

Reply:
Removed this line.